src/pages/resultsView/plots/PlotsTab.tsx

```diff
@@ -0,0 +1,1110 @@
+                    {this.sameGeneInBothAxes && (
+                        <div>
+                            View
+                            <div className="radio"><label>
+                                <input
+                                    type="radio"
+                                    name="utilities_viewMutationType"
+                                    value={EventKey.utilities_viewMutationType}
+                                    checked={this.viewType === ViewType.MutationType}
+                                    onClick={this.onInputClick}
+                                    disabled={!this.mutationDataExists.isComplete || !this.mutationDataExists.result}
+                                /> Mutation Type
+                            </label></div>
+                            <div className="radio"><label>
+                                <input
+                                    type="radio"
+                                    name="utilities_viewMutationType"
+                                    value={EventKey.utilities_viewCopyNumber}
+                                    checked={this.viewType === ViewType.CopyNumber}
+                                    onClick={this.onInputClick}
+                                    disabled={!this.cnaDataExists.isComplete || !this.cnaDataExists.result}
+                                /> Copy Number
+                            </label></div>
+                        </div>
+                    )}
+                    <div style={{marginTop:10}}>
+                        <DownloadControls
+                            getSvg={this.getSvg}
+                            filename={this.downloadFilename}
+                            buttons={["SVG", "PDF"]}
+                            dontFade={true}
+                        />
+                    </div>
+                </div>
+            </div>
+        );
+    }
+
+    private controls() {
+        return (
+            <div style={{display:"flex", flexDirection:"column"}}>
+                <div style={{margin:5, padding:10, border: "1px solid #aaaaaa", borderRadius:4}}>
+                    <Observer>
+                        {this.getHorizontalAxisMenu}
+                    </Observer>
+                </div>
+                <div style={{margin:5}}>
+                    <button onClick={this.swapHorzVertSelections}>Swap Axes</button>
+                </div>
+                <div style={{margin:5, padding:10, border: "1px solid #aaaaaa", borderRadius:4}}>
+                    <Observer>
+                        {this.getVerticalAxisMenu}
+                    </Observer>
+                </div>
+                <div style={{margin:5, padding:10, border: "1px solid #aaaaaa", borderRadius:4}}>
+                    <Observer>
+                        {this.getUtilitiesMenu}
+                    </Observer>
+                </div>
+            </div>
+        );
+    }
+
+    readonly plotType = remoteData({
```

Comment:
we shouldn't have remoteData on a view component.  the reason is that view components can and will be reinstantiated.   remoteData the are properties of component class will then be duplicated, with old ones, I believe never getting cleaned up.     Stores on the other should only be instantiated once (we used to break this rule but in SPA, we have to obey it).

Reply:
i think we already decided that in general it's not pleasing to have to separate things into stores? my opinion is it's better to start explicitly disposing once we do SPA